### PR TITLE
LookupServer: Use pledge()

### PR DESCRIPTION
**Pull request by bgianfo**

### CODE REVIEW

File: Servers/LookupServer/main.cpp [hunk 1]
```diff
@@ -1,13 +1,24 @@
 #include "LookupServer.h"
 #include <LibCore/CEventLoop.h>
+#include <stdio.h>
 
 int main(int argc, char** argv)
 {
     (void)argc;
     (void)argv;
 
+    if (pledge("stdio unix inet cpath rpath fattr ", nullptr) > 0) {
```
Comment (bugaevc) [inline Servers/LookupServer/main.cpp]:
```suggestion
    if (pledge("stdio unix inet cpath rpath fattr", nullptr) < 0) {
```

File: Servers/LookupServer/main.cpp [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+        perror("pledge");
+        return 1;
+    }
+
     CEventLoop event_loop;
     LookupServer server;
 
+    if (pledge("stdio unix inet", nullptr) > 0) {
```
Comment (bugaevc) [inline Servers/LookupServer/main.cpp]:
```suggestion
    if (pledge("stdio unix inet", nullptr) < 0) {
```

File: Servers/LookupServer/main.cpp (hunk 1 above)
Comment (bgianfo) [reply Servers/LookupServer/main.cpp]: Well that's embarrassing :) Thanks for catching!

**Comment by bgianfo** [comment]
Sure, thanks.

**Comment by awesomekling** [comment]
Very nice :)

**Pull request merged**